### ISSUE TITLE
Some logic/formulas from the white paper not been implement in the contract

### ISSUE THREAD

**Issue by ryu9827** [tikonoff/staking-app#15]
White paper page 15:
If Total_Fees > FEE_supply: Generate (Total_Fees - FEE_supply) FEE tokens

We can not find this logic in the contracts. Is this going to be checked and calculated manually? By minter calling sendToken(minter, Fee difference) ?

**Comment by tikonoff** [comment]
```
    uint feeReceived = feeToken.balanceOf(this);
    feeForTheStakingInterval = feeForTheStakingInterval.add(feeReceived.add(this.balance.div(weiPerFee)));
```
No? 
If not -may be it's out of scope?

**Comment by ryu9827** [comment]
I don't think this function have any thing to do with "Generate (Total_Fees - FEE_supply) FEE tokens" .
Maybe they just forget to implement this logic...out of scope

**Comment by tikonoff** [comment]
I think it's pretty close :)
the code above is calculating how many FEE should be generating to the staking interval.
Then:
```
function redeemLevAndFee(address _staker) private validAddress(_staker) isDoneStaking {
    ...
    uint feeEarned = levBlock.mul(feeForTheStakingInterval).div(totalLevBlocks);
    ...
    if (feeEarned > 0) feeToken.sendTokens(_staker, feeEarned);
    ...
  }
```

**Comment by ryu9827** [comment]
updateFeeForCurrentStakingInterval() calculates the feeReceived and then BURN them.
feeToken.sendTokens() is sending FEE tokens from FEE contract to individual users. Even when the minter calls this function, the input "feeEarned" is not the same meaning of "(Total_Fees - FEE_supply) FEE tokens". So, I think this is not implementing the formula. The logic is not right.
